src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -100,24 +100,31 @@ export const getThorTradeQuote = async (
     sellAmountCryptoBaseUnit,
     receiveAddress,
     affiliateBps,
-    slippageBps,
   })
 
   if (maybeQuote.isErr()) return Err(maybeQuote.unwrapErr())
 
   const thornodeQuote = maybeQuote.unwrap()
-  const {
-    slippage_bps: recommendedSlippageBps,
-    fees,
-    expected_amount_out: expectedAmountOutThorBaseUnit,
-    memo,
-  } = thornodeQuote
-
-  const recommendedSlippageDecimalPercentage = convertBasisPointsToDecimalPercentage(
-    recommendedSlippageBps.toString(),
-  ).toString()
-
-  const rate = (() => {
+  const { fees } = thornodeQuote
+
+  const perRouteValues = [
+    {
+      // regular swap
+      source: SwapperName.Thorchain,
+      slippageBps: thornodeQuote.slippage_bps,
+      expectedAmountOutThorBaseUnit: thornodeQuote.expected_amount_out,
+      isStreaming: false,
+    },
+    {
+      // streaming swap
+      source: `${SwapperName.Thorchain} • Streaming` as SwapSource,
+      slippageBps: thornodeQuote.streaming_slippage_bps,
+      expectedAmountOutThorBaseUnit: thornodeQuote.expected_amount_out_streaming,
+      isStreaming: true,
+    },
+  ]
```

Comment:
Given this comment `Note: information about thorchain streaming swaps for users will be added in a follow-up PR and is out of scope for the first release.` not sure if querying for actual streaming quotes should be implemented in this PR or a follow-up, but we will also need to pass streaming-related queryparams in addition to making a memo with streaming values, as currently both quotes will be the same, since streaming is effectively disabled:

<img width="492" alt="image" src="https://github.com/shapeshift/web/assets/17035424/9d104606-ce08-407e-a0ba-751a5a62e89e">